static/bidder-info/edge226.yaml

```diff
@@ -0,0 +1,20 @@
+endpoint: "http://ssp.dauup.com/pserver"
+maintainer:
+  email: "audit@edge226.com "
```

Comment:
Please remove the extra space after `"audit@edge226.com "`. It should be `"audit@edge226.com"`